frontend/js/annotation-tool.js

```diff
@@ -731,9 +731,16 @@ define(["jquery",
 
                 bookData.forEach(function (arr) {
                     Object.keys(arr).forEach(function (key) {
-                        var value = arr[key] === null ? '' : arr[key];
+                        let value = arr[key] === null ? '' : arr[key];
```

Comment:
```suggestion
                        var value = arr[key] === null ? '' : arr[key];
```

I know you like `let`, and I do agree with you (well, kind of; this variable could actually be `const`, even), but the consistent thing to do in this code base is unfortunately still `var`.

Some day we will get there, I hope ... :wink: